target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -404,6 +404,11 @@ wrh-300cr)
 xzwifi,creativebox-v1)
 	ucidef_set_led_switch "internet" "internet" "$boardname:blue:internet" "switch0" "0x10"
 	;;
+x1)
+	ucidef_set_led_default "system" "system" "green:system" "0"
+	ucidef_set_led_netdev  "wwan_up" "MobileDataUp" "blue:lteStatus" "wwan0" "link tx rx"
```

Comment:
Look at my LED label comments in your DTS file, and try to name the LEDs accordingly like `x1:blue:lte-online` and `x1:red:lte-offline` or something like that.